src/Settings/Settings/SettingsAll.php

```diff
@@ -179,10 +162,12 @@ private function getAllSettingsSidebars(): array
 	 */
 	private function getAllSettings(): array
 	{
-		$allSettings = self::ALL_SETTINGS;
+		$allSettings = [];
 
-		foreach (Integrations::ALL_INTEGRATIONS as $key => $integration) {
-			$allSettings[$key] = $integration['settings'] ?? '';
+		foreach (Filters::ALL as $key => $integration) {
+			if (isset($integration['settings'])) {
+				$allSettings[$key] = $integration['settings'] ?? '';
+			}
```

Comment:
Same as the above comment. Or if you don't need to set it to empty, you can just use `continue` inside the foreach to skip the integrations that aren't set.